addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -29,18 +29,16 @@ def _default_user_ids(self):
         return [(6, 0, [self.env.uid])]
 
     def _default_button_text(self):
-        return _('Have a Question? Chat with us.')
+        return _('Need help? Chat with us.')
 
     def _default_default_message(self):
         return _('How may I help you?')
 
     # attribute fields
     name = fields.Char('Channel Name', required=True)
-    button_text = fields.Char('Text of the Button', default=_default_button_text,
-        help="Default text displayed on the Livechat Support Button", translate=True)
+    button_text = fields.Char('Text of the Button', default=_default_button_text, translate=True)
     default_message = fields.Char('Welcome Message', default=_default_default_message,
         help="This is an automated 'welcome' message that your visitor will see when they initiate a new conversation.", translate=True)
-    input_placeholder = fields.Char('Chat Input Placeholder', help='Text that prompts the user to initiate the chat.', translate=True)
```

Comment:
Just a note: if we miss the freeze with this one, we can re-target and keep the fields in 19.0 but stop using them.
And target the upgrade for 19.1.